### ISSUE TITLE
Распродажа

### ISSUE THREAD

**Issue by KaneNOD** [yegorf1/RogueBot#173]
Редкая комната-ивент, позволяет получить любой из предметов алхимика со скидкой до 60%(скидка генерируется случайным образом при попадании в комнату)

**Comment by thekiba** [comment]
Можно реализовать комнату, в которой ты натыкаешься на предложение конкретного предмета со скидкой. Купить его можно в единичном экземпляре.

**Comment by KaneNOD** [comment]
`На входе в комнату ты видишь огромную надпись:"Не более двух предметов в одни руки!`
может так?

**Comment by thekiba** [comment]
Хм.. Два предмета по цене одного или как?

**Comment by KaneNOD** [comment]
нет, просто максимум сколько можно купить предметов одного типа - 2
скидка генерируется

**Comment by Fadds** [comment]
Не забудьте округлить до целого :)
